hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/contract/ContractCallTransitionLogic.java

```diff
@@ -118,7 +129,8 @@ public void doStateTransitionOperation(
         final var target = targetOf(op);
         final var targetId = target.toId();
         Account receiver;
-        if (target.equals(EntityNum.MISSING_NUM)
+        final var isMissing = target.equals(EntityNum.MISSING_NUM);
```

Comment:
I would have gone ahead and called this `targetIsMissing` (because, not `sender` or `receiver`.)

Reply:
Renamed to `targetAddressIsMissing`.